control/bgp.yaml

```diff
@@ -0,0 +1,83 @@
+openapi: 3.0.3
+info:
+  title: BGP Control models
+  description: >-
+    All BGP control schemas
+  version: ^0.0.0
+components:
+  schemas:
+    BGP.Notification:
+      description: >-
+        A NOTIFICATION message is sent when an error is detected with the BGP session, such as hold timer expiring, misconfigured AS number 
```

Comment:
Would suggest rephrasing.

```
Send explicit NOTIFICATION for a list of specified BGP peers. A user can send IANA defined BGP NOTIFICATION according to https://www.iana.org/assignments/bgp-parameters/bgp-parameters.xhtml. 

A NOTIFICATION message is sent when an error with a BGP session is detected, such as hold timer expiring, misconfigured AS number or a BGP session reset requested. This causes the BGP connection to close. 
```

NOTE, I replaced "user defined" with "explicit" as most of them are IANA defined (and hence not user defined)

Reply:
updated